React/wbooks/src/services/bookService.js

```diff
@@ -2,14 +2,8 @@ import api from '../app/config/api';
 
 import * as localStorage from './localstorageService.js';
 
-export const retrieveUserFromSession = () => {
-  return localStorage.retrieveUserTokenFromLocalStorage();
-}
+export const retrieveUserFromSession = () => localStorage.retrieveUserTokenFromLocalStorage();
```

Comment:
Why are u repeating this functions in both services? Just import the local storage services in the file what u need